hooks/useMousePosHook.tsx

```diff
@@ -1,27 +1,43 @@
 import { useEffect, useState } from "react";
 
 function useMousePosHook(ref: React.RefObject<any>) {
-  const [isInside, setIsInside] = useState(false);
+  const [isHoverInside, setIsHoverInside] = useState(false);
 
-  function handleClickOutside(event: MouseEvent) {
+  function handleHover(event: MouseEvent) {
```

Comment:
ayaw to i delete ang `handleClickOutside` nga function, i add lang ni ang `handleHover`, so basically naa tay handleClickOutside AND handleHover

Reply:
resolved